content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -550,14 +585,16 @@ restrictions around nodes: pods from several namespaces may run on the same node
 
 ## Limit Priority Class consumption by default
 
-It may be desired that pods at a particular priority, eg. "cluster-services", should be allowed in a namespace, if and only if, a matching quota object exists.
+It may be desired that pods at a particular priority, eg. "cluster-services", 
+should be allowed in a namespace, if and only if, a matching quota object exists.
 
-With this mechanism, operators will be able to restrict usage of certain high priority classes to a limited number of namespaces and not every namespace will be able to consume these priority classes by default.
+With this mechanism, operators will be able to restrict usage of certain high
+priority classes to a limited number of namespaces and not every namespace
```

Comment:
nit: This section is a bit unclear, but perhaps this section can be updated in a future PR.
nit: With this mechanism, operators are able to restrict usage ...

Reply:
OK